src/js/reducers.js

```diff
@@ -0,0 +1,27 @@
+import { combineReducers } from 'redux';
+
+const initialState = {
+  currentPatch: {
+    name: 'RolandJuno106',
+    state: {
+      lfo: 0
+    }
+  }
+};
+
+function PatchAppReducer(state = initialState, action) {
+  switch (action.type) {
+  case 'SET_PARAM':
+    return Object.assign({}, state, {
+      currentPatch: {
+        state: {
+          [action.path]: action.value
```

Comment:
@dbalatero scope this [computed property name](http://es6-features.org/#ComputedPropertyNames) action